src/components/modules/donations/list/filters/DonationsFilter.tsx

```diff
@@ -160,6 +174,21 @@ export const DonationsFilterComponent: React.FunctionComponent = () => {
             }}
           ></EffektCheckForm>
         </FilterGroup>
+
+        <FilterGroup>
+          <FilterGroupHeader>Organizations</FilterGroupHeader>
```

Comment:
Not sure if the choice of name on UI is correct. Suggestions welcome for better one.

Reply:
This is good :+1: